Resources/Prototypes/_Goobstation/GameRules/Dynamic/base.yml

```
@@ -34,4 +34,4 @@
   - type: AntagLoadProfileRule
     speciesOverride: Human
     speciesOverrideBlacklist:
-    - Vox
+    - Vox # lol loser
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 5

(indentation)